theme

```diff
@@ -66,4 +66,5 @@
   "tgcowell/waves",
   "williamahartman/noctis-solarized",
   "wowgamr/animated-weather-card"
```

Comment:
I you have to include a coma `,` at the end of this line. The Checks are failing since the json structure is a little off 👍

Reply:
Thanks for letting me know what I had messed up! It's fixed now.